app/src/main/AndroidManifest.xml

```diff
@@ -51,31 +47,33 @@
       android:value="@integer/google_play_services_version"/>
     <meta-data
       android:name="com.google.android.maps.v2.API_KEY"
-      android:value="@string/google_maps_key"/>
+      android:value="AIzaSyA8R2PX8tveUxoamwEk0yh1pn-ACEU7L6Q"/>
```

Comment:
this key needs to be removed eventually because it has been released to public. We would need a new key value pair before releasing, and this key will have to invalidated.